pkg/security/security_profile/profile/manager.go

```diff
@@ -550,6 +550,20 @@ func (m *SecurityProfileManager) SendStats() error {
 			profileStats[profile.Status] = make(map[bool]float64)
 		}
 		profileStats[profile.Status][profile.loadedInKernel]++
+
+		uptime := uint64(0)
+		if !profile.loadedInKernel {
+			uptime = (profile.unloadedNano - profile.loadedNano) / 1000000000
```

Comment:
i believe you can replace 1000000000 with time.Second

Reply:
suggestion applied, thanks!